cmd/gitlab.go

```diff
@@ -386,86 +362,145 @@ func (c *Client) pollPipelineJobs(p Project, pipelineID int, topics string, ref
 	return err
 }
 
-func (c *Client) pollProjectRef(p Project, ref string) {
-	var gp *gitlab.Project = p.GitlabProject
-	topics := strings.Join(gp.TagList[:], ",")
+func (c *Client) pollProjectRef(p Project, ref string) error {
+	var gp = p.GitlabProject
 	var lastPipeline *gitlab.Pipeline
+	topics := strings.Join(gp.TagList[:], ",")
 
+	//FIXME is adding the counter a 0 value at every run a problem?
 	runCount.WithLabelValues(gp.PathWithNamespace, topics, ref).Add(0)
+	log.Debugf("Polling %v:%v (%v)", gp.PathWithNamespace, ref, gp.ID)
 
-	b := &backoff.Backoff{
-		Min:    time.Duration(cfg.PipelinesPollingIntervalSeconds) * time.Second,
-		Max:    time.Duration(cfg.PipelinesMaxPollingIntervalSeconds) * time.Second,
```

Comment:
we should remove this value from the general config as it is not used anymore

Reply:
I removed the backoff because I could not see any value in having it: the clients already have a rateLimiting so could not really undertand its use. If it's ok for you to removing it, I'll fix that part of the doc too

Reply:
Yes, this was implemented before the client has such a mechanism in place 👌 

You can remove it from the docs!

Reply:
thanks, removed in latest commit: from README, from Config and from examples and tests

Reply:
thanks!